app/components/TransferModal.tsx

```diff
@@ -166,30 +174,46 @@ export const TransferModal = ({
 
   const tokenBalance = Number(smartWalletBalance?.balances[token]) || 0;
 
-  const renderSuccessView = () => (
-    <div className="space-y-6 pt-4">
-      <CheckmarkCircle01Icon className="mx-auto size-10" color="#39C65D" />
-
-      <div className="space-y-3 pb-5 text-center">
-        <h2 className="text-lg font-semibold text-text-body dark:text-white">
-          Transfer successful
-        </h2>
+  const renderSuccessView = () => {
+    const explorerLink = transactionHash
+      ? getExplorerLink(selectedNetwork.chain.name, transactionHash)
+      : null;
+
```

Comment:
the resultant link will be invalid... client?.sendTransaction returns the userop hash not the final txhash

Reply:
done. changes effected.